common/src/main/java/com/wynntils/wynn/item/IntelligenceSkillPointsItemStack.java

```diff
@@ -62,6 +62,9 @@ private List<Component> getManaTables() {
         for (int j = 0; j < 4; j++) {
             SpellType spell = SpellType.forClass(characterInfo.getClassType(), j + 1);
 
+            if (spell == null) {
```

Comment:
This does fix the NPE but doesn't fix `spell` being `null`. Which shouldn't be as we have fail over spells with class `None`.